incubator/hnc/config/crd/bases/hnc.x-k8s.io_hierarchyconfigurations.yaml

```diff
@@ -91,13 +91,12 @@ spec:
                       when the HNC controller is restarted. These should quickly resolve
                       themselves (<30s). \n Currently, the supported values are: \n
                       - \"CritParentMissing\": the specified parent is missing \n
-                      - \"CritParentInvalid\": the specified parent is invalid (e.g.,
-                      would cause a cycle) \n - \"CritAncestor\": a critical error
-                      exists in an ancestor namespace, so this namespace is no longer
-                      being updated either. \n - \"SubnamespaceAnchorMissing\": this
-                      namespace is a subnamespace, but the anchor referenced in its
-                      `subnamespaceOf` annotation does not exist in the parent. \n
-                      - \"CannotPropagateObject\": this namespace contains an object
+                      - \"CritCycle\": the namespace is a member of a cycle \n - \"CritAncestor\":
```

Comment:
nit: The term `cycle` could be confusing to many people. I'm not sure the typical sysadmin would understand what this means given the context.

Reply:
Done, PTAL